docs/protocol/network/validators.md

```diff
@@ -4,15 +4,13 @@ title: Validators
 description: "Learn about NEAR Protocol validators, their roles in network security, consensus mechanisms, validator economics, and how to become a validator."
 ---
 
-The NEAR network is decentralized, meaning that multiple people collaborate in order to keep it safe. We call such people **validators**.
+At its core, NEAR Protocol is a decentralized blockchain that operates on a network of independent participants called validators who process transactions and secure the network.
 
-In order to make sure that all the transactions in the network are valid, i.e. that nobody is trying to steal money, the validators follow a specific consensus
-mechanism.
+These validators must reach consensus—meaning they need to agree on which transactions are valid and should be added to the blockchain. This ensures no one can steal funds, double-spend tokens, or manipulate the system. The method validators use to reach this agreement is called a consensus mechanism.
 
-Currently, there are a few well-known consensus mechanisms to keep a blockchain working correctly and resistant to attacks.
-NEAR Protocol uses a version of **Proof-of-Stake**, particularly [Thresholded Proof of Stake](https://near.org/blog/thresholded-proof-of-stake/).
+NEAR Protocol uses [Proof-of-Stake (PoS)](https://en.wikipedia.org/wiki/Proof_of_stake), a consensus mechanism that secures the network through economic incentives rather than energy-intensive computational power. Unlike Bitcoin's [proof-of-work](https://en.wikipedia.org/wiki/Proof_of_work) system that requires massive amounts of electricity, PoS makes the network environmentally sustainable while maintaining robust security.
```

Comment:
```suggestion
NEAR validators use [Proof-of-Stake (PoS)](https://en.wikipedia.org/wiki/Proof_of_stake), a consensus mechanism that secures the network through economic incentives rather than energy-intensive computational power. Unlike Bitcoin's [proof-of-work](https://en.wikipedia.org/wiki/Proof_of_work) system that requires massive amounts of electricity, PoS makes the network environmentally sustainable while maintaining robust security.
```